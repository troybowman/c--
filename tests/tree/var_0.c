void main(char argv[]) { x }
