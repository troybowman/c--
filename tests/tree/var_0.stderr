error: use of undeclared identifier x at line 1
syntax error, line: 1
